nms/app/packages/magmalte/app/views/metrics/Explorer.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+import ActionTable from '../../components/ActionTable';
+import CardTitleRow from '../../components/layout/CardTitleRow';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import ExploreIcon from '@material-ui/icons/Explore';
+import Grid from '@material-ui/core/Grid';
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+import React from 'react';
+import nullthrows from '@fbcnms/util/nullthrows';
+import useMagmaAPI from '@fbcnms/ui/magma/useMagmaAPI';
+import type {prometheus_labelset} from '@fbcnms/magma-api';
+
+import {colors, typography} from '../../theme/default';
+import {makeStyles} from '@material-ui/styles';
+import {useEffect, useState} from 'react';
+import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+const TITLE = 'Metrics Explorer';
+
+const useStyles = makeStyles(theme => ({
+  dashboardRoot: {
+    margin: theme.spacing(5),
+  },
+  appBarBtn: {
+    color: colors.primary.white,
+    background: colors.primary.comet,
+    fontFamily: typography.button.fontFamily,
+    fontWeight: typography.button.fontWeight,
+    fontSize: typography.button.fontSize,
+    lineHeight: typography.button.lineHeight,
+    letterSpacing: typography.button.letterSpacing,
+
+    '&:hover': {
+      background: colors.primary.mirage,
+    },
+  },
+  appBarBtnSecondary: {
+    color: colors.primary.white,
+  },
+}));
+
+export type MetricsDetail = {
+  MetricName: string,
+  PromQL: string,
+  Description: string,
+  Category: string,
+  Service: string,
+};
+
+export default function MetricsExplorer() {
+  const classes = useStyles();
+  const [isLoading, setIsLoading] = useState(true);
+  const [LteMetricsTable, setLteMetricsTable] = useState([]);
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const {match} = useRouter();
+
+  const networkId = nullthrows(match.params.networkId);
+  const {
+    response: metricSeries,
+    isLoading: isMetricSeriesLoading,
+  } = useMagmaAPI(MagmaV1API.getNetworksByNetworkIdPrometheusSeries, {
```

Comment:
One thing to note is that this API endpoint is pretty slow on large deployments. You might want to pass in a start/end time of 1 hour or something to significantly decrease the time it takes to load. There's a small chance you'll miss out on some metrics but it's fairly unlikely as the series don't change very much at all.